GVFS/GVFS.Common/Maintenance/LooseObjectsStep.cs

```diff
@@ -148,17 +230,24 @@ protected override void PerformMaintenance()
                     }
 
                     int beforeLooseObjectsCount = this.CountLooseObjects();
-                    GitProcess.Result result = this.RunGitCommand((process) => process.PrunePacked(this.Context.Enlistment.GitObjectsRoot), nameof(GitProcess.PrunePacked));
+                    GitProcess.Result gitResult = this.RunGitCommand((process) => process.PrunePacked(this.Context.Enlistment.GitObjectsRoot), nameof(GitProcess.PrunePacked));
                     int afterLooseObjectsCount = this.CountLooseObjects();
 
-                    int objectsAddedToPack = this.CreateLooseObjectsPackFile();
+                    CreatePackResult result = this.TryCreateLooseObjectsPackFile(out int objectsAddedToPack);
 
                     EventMetadata metadata = new EventMetadata();
                     metadata.Add("GitObjectsRoot", this.Context.Enlistment.GitObjectsRoot);
                     metadata.Add("StartingCount", beforeLooseObjectsCount);
                     metadata.Add("EndingCount", afterLooseObjectsCount);
                     metadata.Add("RemovedCount", beforeLooseObjectsCount - afterLooseObjectsCount);
                     metadata.Add("LooseObjectsPutIntoPackFile", objectsAddedToPack);
+                    metadata.Add("CreatePackResult", result.ToString());
+
+                    if (result == CreatePackResult.CorruptBlob)
+                    {
+                        int removedObjects = this.ClearCorruptLooseObjects(metadata);

```

Comment:
Do we need this method to return anything if we aren't going to use it?  We already added the information to the `metadata`.